spec/factories/factories_user.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :user do
+    first_name 'Tony'
+    last_name 'Stark'
+    email 'tonystark@example.com'
+    phone '+380310000000'
+    gender true
+    date_of_birth DateTime.new(1990, 1, 1)
+    admin false
```

Comment:
please define inside 'factory :user do' :
 ```
factory :admin do
      admin true
    end
```
it make easy to create admin user for tests using `create :admin`